var.mk

```diff
@@ -33,8 +33,9 @@ STACK_SETUP := $(shell $(STACK) setup > /dev/null)
 STACK_INSTALL_DEPENDENCIES := $(shell $(STACK) build --only-dependencies $(STACK_DEPENDENCIES_FLAGS) > /dev/null)
 
 GHCVERSION := $(shell $(STACK_EXEC) ghc --numeric-version)
-ifneq ($(findstring 7., $(GHCVERSION)),)
-GHC7RTSOPTS := -rtsopts
+GHC_RTSOPTS_AVAILABLE := $(shell expr $(shell $(STACK_EXEC) ghc --numeric-version | cut -d '.' -f 1) \>= 7)
```

Comment:
What does this 'stack exec --' buy here?
Also it is too complicated and clutters the Makefile w/o reason. Using `VAR += $(shell [ _cond_ ] && echo _opts2add_)` is better. 
last but not least since it gets touched anyway: it can be made more efficient and correct: e.g. with `GHCRTSOPTS := $(shell X=$(GHCVERSION) && [ $${X%%.*} -gt 7 ] && echo '-rtsopts')`